crates/logic-test/Cargo.toml

```diff
@@ -5,12 +5,17 @@ edition.workspace = true
 authors.workspace = true
 
 [dependencies]
-gluesql = { version = "0.13", git = "https://github.com/gluesql/gluesql.git", rev = "91d57c4f0c87a638fc3aacba50caadeabc9e0302" }
+async-trait = "0.1"
+gluesql = { git = "https://github.com/gluesql/gluesql.git", rev = "a361d9a6c62bf085d99278ddc47490afec78f973" }
 # depend on gluesql-core
-chrono = "=0.4.23"
+chrono = "=0.4.41"
```

Comment:
https://github.com/gluesql/gluesql/blob/a361d9a6c62bf085d99278ddc47490afec78f973/core/Cargo.toml#L20
Since the chrono version is not fixed, it seems like the '=' can be removed.